src/Testura.Code/Saver/CodeSaver.cs

```diff
@@ -81,4 +109,13 @@ private AdhocWorkspace CreateWorkspace()
 
         return cw;
     }
+
+    private void EnsurePathExists(string filePath)
+    {
+        var fi = new FileInfo(filePath);
```

Comment:
Minor thing but should maybe use the full name for FileInfo variables? So ` var fileInfo = new FileInfo(path)`.

I just take it's easier to read and understand when we use clear names for a variable (which may sound a bit like hypocrisy as I named CompilationUnitSyntax `cu`...). But it's more about taste and if you prefer fi we can stick with that.

Reply:
Sounds good! I am always for more 'explicit' style of programming, even if its more verbose - its easier to understand.

Haha, yea we all get lazy at times, thats why I use `fi`, its just laziness haha. I'll make the change now

Reply:
Done! I changed a few just to make them more explicit. I'd love to do this everywhere in the code eventually.

I'd also love to add more constructors and such so we can avoid using `null` as a parameter Such as this example:

```csharp
var builder = new MethodBuilder("MyMethod").WithBody(null).Build()

It will create this:

void MyMethod();
```

I won't do that in this PR  - but I just wanted to mention it as its related to the topic of being more explicit

Reply:
Sounds like a good idea to remove withbody requirment. Approved the PR as well.